write-fonts/src/tables/variations/ivs_builder.rs

```diff
@@ -46,21 +49,34 @@ pub struct VariationIndexRemapping {
 struct DeltaSet(Vec<(u16, i32)>);
 
 impl VariationStoreBuilder {
-    pub fn add_deltas<T: Into<i32>>(&mut self, deltas: Vec<(VariationRegion, T)>) -> u32 {
+    /// Create a builder that will optimize delta storage
+    pub fn new() -> Self {
```

Comment:
Nit: I might prefer optimized for symmetry with unoptimized.

Reply:
maybe add both methods and have default new constructor call optimized

Reply:
then VariationStoreBuilder would have three constructors that do exactly the same thing, syntactic sugar addicted

Reply:
yea I went back and forth on this, e.g. considered removing `default` and only having explicit constructors, then didn't want to break existing code, then wanted names that made it clear that one of the constructors really *is* preferred in the general case, then realized this was also a good argument for default... 🥴 

I also don't like `optimized` since the tense is wrong. I hate all options equally?

Reply:
okay I've gone with `new` and `new_with_implicit_indices`.